recipes/proj/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
If the reason you're adding `dl` because sqlite3 requires it, you can modify `conan_basic_setup()` to `conan_basic_stup(TARGETS)` and modify `src/lib_proj.cmake` to let link `${PROJ_CORE_TARGET}` to `CONAN_PKG::sqlite3` instead of `${SQLITE3_LIBRARY}`.

Reply:
Thanks for pointing that out.